README.md

```diff
@@ -45,8 +45,40 @@ The goal is to have all supported backends at feature parity, but until then, co
 
 Contributions that fill in the gaps are welcome!
 
-Running tests
-=============
+Development
+===========
+
+### First-time installation of protobuf and other codegen tools
+
+You need to install and run the protobuf compiler before you can regenerate Go code after you change the `vcs.proto` file.
+
+1.	**Install protoc**, the protobuf compiler. Find more details in the [protobuf README](https://github.com/google/protobuf).
+
+	```
+	brew install --devel protobuf
```

Comment:
Should we apply the same change at https://github.com/sourcegraph/go-sourcegraph/blob/nodb/README.md#first-time-installation-of-protobuf-and-other-codegen-tools?